selftests/functional/test_basic.py

```diff
@@ -80,6 +80,18 @@ def test_my_name(self):
 '''
 
 
+VALID_PYTHON_TEST_WITH_TAGS = '''
+from avocado import Test
+
+class MyTest(Test):
+    def test(self):
+         """
+         :avocado: tags=BIG_TAG_NAME
+         """
+         pass
```

Comment:
One thing I don't understand is why do you keep adding new in-code tests when we already have several example tests including tags and we do use them in selftests... Anyway it works well and I know you prefer that.